src/backend/marsha/core/defaults.py

```diff
@@ -118,8 +118,8 @@
     (CC_BY_SA, _("Creative Common By Attribution Share Alike")),
     (CC_BY_NC, _("Creative Common By Attribution Non Commercial")),
     (CC_BY_NC_SA, _("Creative Common By Attribution Non Commercial Share Alike")),
-    (CC_BY_ND, _("Creative Common By Attribution No Derivates")),
-    (CC_BY_NC_ND, _("Creative Common By Attribution Non Commercial No Derivates")),
+    (CC_BY_ND, _("Creative Common By Attribution No Derivatives")),
+    (CC_BY_NC_ND, _("Creative Common By Attribution Non Commercial No Derivatives")),
```

Comment:
I have checked online, is it the right [license](https://pitt.libguides.com/openlicensing/cc-noderivatives#:~:text=The%20Creative%20Commons%20No%20Derivatives,the%20more%20restrictive%20license%20conditions.)?

Reply:
Yes it is: https://creativecommons.org/licenses/by-nc-nd/4.0/